types/command/command.go

```diff
@@ -89,7 +89,7 @@ const (
 const (
 	// JobPhaseCompleted represents a Kubernetes Job with
 	// Completed status
-	JobPhaseCompleted string = "Completed"
+	JobPhaseCompleted string = "Complete"
```

Comment:
I think there are edge cases. We should try to write the integration tests & see how things work.
We can team up with @amitbhatt818 or others if you want.

Reply:
I am on it writing integration test cases

Reply:
As per code flow, I am not seeing any edge case except deletion & creation in the same reconciliation